src/spikeinterface/curation/curation_tools.py

```diff
@@ -133,3 +134,44 @@ def find_duplicated_spikes(
         return _find_duplicated_spikes_keep_last_iterative(spike_train.astype(np.int64), censored_period)
     else:
         raise ValueError(f"Method '{method}' isn't a valid method for find_duplicated_spikes. Use one of {_methods}")
+
+
+def remove_empty_units(sorting_or_sorting_analyzer, minimum_spikes=10):
```

Comment:
We already have this as a method Sorting no ?
I would put 0 by default.
We should make a doc

Reply:
removed